CONTRIBUTING.md

```diff
@@ -2,106 +2,136 @@
 
-##### Version definitions
+#### Version definitions
 
-**Major.minor.patch**
 
-* Major
-  * openstack release
-  * possibly feature?
-* minor
-  * big change within release
-  * security  - upgrade-impacting
-  * bug fixes - upgrade-impacting
-  * features
+**Major release**
+
+  * OpenStack release
+  * Possibly a new feature (?)
+
+**Minor release**
+  
+  * Big change within a release
+  * Security  - upgrade-impacting
+  * Bug fixes - upgrade-impacting
+  * Features
   * sha bumps
-* patch
-  * security  - non-upgrade-impacting
-  * bug fixes - non-upgrade-impacting
+   
+**Patch release**
+  
+  * Security  - non-upgrade-impacting
+  * Bug fixes - non-upgrade-impacting
 
 ## Working in the repository
 
-### filing a bug report
+### Filing a bug report
+
+File a bug or requested feature against ```rpc-openstack```. Please include the following information:
 
-When creating an issue, please ensure you include the following information
+* The branch (or tag) which you encountered this issue.
+* Steps to reproduce the issue.
+* Any output or logs from your environment.
 
-* branch (or tag) on which you encountered this issue
-* Steps to reproduce
-* Any output or logs that you gathered from your environment
+### Fixing a bug
 
-### fixing a bug
 
-1. A bug is filed or feature requested against ```rpc-openstack```
-2. After triage and prioritisation, the developer pushes a fix directly to a branch on ```rpc-openstack``` in the ```rcbops``` github namespace (rather than a developers own fork). A pull request is then made from this branch to the ```master``` branch.
-3. Unless a bug is specific to a release branch (e.g. ```liberty-12.1```), fixes are made to the ```master``` branch before any potential backports.
+1. After the bug is triaged and prioritised, make the required changes. New features, breaking changes and other patches of note must include a release note generated using the `reno tool`. Please see `Release Notes` for more information.
+2. Push changes directly to a branch on ```rpc-openstack``` in the ```rcbops``` github namespace (rather than a developers own fork). A pull request is then made from this branch to the ```master``` branch.
+3. Unless a bug is specific to a release branch (e.g. ```liberty-12.1```), commit fixes to the ```master``` branch before any potential backports.
 4. Bugs meeting backport criteria are backported to the release branches (e.g. ```liberty-12.1```) as appropriate.
-5. github markdown is used to update the original issue with a checklist for tracking which branches have had fixes merged.
+5. Github markdown is used to update the original issue with a checklist for tracking which branches have fixes merged.
 6. Each time a PR is merged, the associated branch is deleted from ```rpc-openstack```.
 7. When all PRs are completed the issue is then closed.
 
 
-## Commits, pull requests and reviews
+## Commits, Pull Requests and Release Note Guidelines
 
```

Comment:
The document order goes 'Commits, Release Note Guidelines, Pull Requests'. I'd suggest either this heading should change to match, or the doc flow change to match the heading.